tests/test_popularity_recommender.py

```diff
@@ -0,0 +1,77 @@
+"""Test Song Recommendation functions."""
+import pytest
+
```

Comment:
As for the format, we want to have two blank lines after that, and for this, we do not necessarily need pytest to get imported. Have updated in this thread